proj/hps_accel/Makefile

```diff
@@ -75,13 +75,19 @@ DEFINES += INCLUDE_MODEL_HPS
 # Use PLATFORM=common_soc TARGET=digilent_arty to select Arty instead of hps
 PLATFORM=hps
 
+# Parallel software and bitstream build
+PARALLEL ?= no
```

Comment:
Having this variable `PARALLEL` is confusing since below we pass an option `--parallel` in `EXTRA_LITEX_ARGS` (which is the option to spawn `nproc` nextpnr jobs).   Let's use something different here like "PARALLEL_BUILD".

Reply:
I have actually removed this variable as it is not required. I used it only to enable the extra nextpnr parameters